source/pervasive/std_specs/core.rs

```diff
@@ -9,4 +9,17 @@ pub fn ex_swap<T>(a: &mut T, b: &mut T)
     core::mem::swap(a, b)
 }
 
+#[verifier(external_type_specification)]
+pub struct ExOption<V>(core::option::Option<V>);
+
+#[verifier(external_type_specification)]
+pub struct ExResult<T, E>(core::result::Result<T, E>);
+
+#[verifier(external_type_specification)]
+pub struct ExRange<Idx>(core::ops::Range<Idx>);
+
+#[verifier(external_type_specification)]
+#[verifier(external_body)]
```

Comment:
Does `external_body` add anything here?  `Duration` already has no public fields.

Reply:
actually, the only reason I added this at all was that I wanted to test this particular combination of features in a cross-crate context